protoc-gen-openapiv2/internal/genopenapi/template_test.go

```diff
@@ -30,6 +30,12 @@ import (
 
 var marshaler = &runtime.JSONPb{}
 
+func assertEqual(t *testing.T, testName string, a interface{}, b interface{}) {
```

Comment:
I appreciate the intention here, but this pattern is inconsistent with the rest of the code, and I don't think we should introduce it here. If you want to do this, we'll need one function for each type you want to test:
```go
func assertEqualInt(t *testing.T, testName string, a int, b int) {
```